cmake/stm32/utilities.cmake

```diff
@@ -42,7 +42,7 @@ endfunction()
 
 include(FetchContent)
 
-set(STM32_FETCH_FAMILIES       F0      F1      F2      F3      F4      F7      G0      G4      H7      L0      L1      L4      L5)
+set(STM32_FETCH_FAMILIES       F0      F1      F2      F3      F4      F7      G0      G4      H7      L0      L1      L4      L5 )

```

Comment:
Possible to add fetch MP1 ?
+
Could you add an MP1 action for automatic build ? (We can do this in a different PR if you are not ready to do it)